### PR TITLE
[exporter/awscloudwatchlogs] Enable component

### PR DESCRIPTION
**Pull request by aateeqi**
**Link to tracking Issue:** 

https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/6991

**Testing:** <Describe what testing was performed and which tests were added.>

**Documentation:** <Describe the documentation added.>

### CODE REVIEW

**Review by Aneurysm9 — APPROVED** [review]
Please add a CHANGELOG entry noting that this component is now available for use.

**Comment by jpkrohling** [comment]
Make sure as well to add this to the -releases repository: https://github.com/open-telemetry/opentelemetry-collector-releases

**Comment by aateeqi** [comment]
@jpkrohling Hey Juraci, what should be done in the releases repo regarding this change?

**Comment by jpkrohling** [comment]
Include this component in the distribution's manifest: https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml

**Review by jpkrohling — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -41,6 +41,7 @@
 - `scrapertest`: Add comparison option to ignore specific attributes (#6519)
 - `tracegen`: Add option to pass in custom headers to export calls via command line (#7308)
 - `tracegen`: Provide official container images (#7179)
+- `awscloudwatchlogsexporter`: enable awscloudwatchlogsexporter which accepts and exports log data (#7297)
```
Comment (jpkrohling) [inline CHANGELOG.md:44]: 0.43.0 was just tagged, you might want to move this to "unreleased".

**Review by jpkrohling — APPROVED** [review]

**Pull request merged**